websites/L/Library Of Babel/presence.ts

```diff
@@ -0,0 +1,120 @@
+const presence = new Presence({
+	clientId: "1212508565802192936",
+});
+
+// Other default assets can be found at index.d.ts
+const enum Assets {
+	Logo = "https://libraryofbabel.info/img/hex802.png",
+	smallImageKey = "",
+}
+
+const enum Pages {
+	homepage = "/",
+	about = "/about.html",
+	search = "/search.html",
+	sRes = "/search.cgi",
+	book = "/book.cgi",
+	browse = "/browse.cgi",
+	bookmark = "/bookmark.cgi",
+	anglishized = "/anglishize.cgi",
+	theory = "/theory.html",
+	images = "/slideshow.html",
+	imgsearch = "/imagesearch.html",
+	imgSRes = "/imagesearch.cgi",
+	coloroscopy = "/coloroscopy.html",
+	refhex = "/referencehex.html",
+}
+
+presence.on("UpdateData", async () => {
+	if (!document.location.toString()) return;
```

Comment:
This condition is likely always false?

Reply:
I added this to get past a weird this firefox does when it redirects to `about:blank`